### ISSUE TITLE
laminasuser (resolves to invalid controller class or alias: laminasuser)

### ISSUE THREAD

**Issue by jmpavan** [fgsl/LaminasUser#4]
I have done the installation following the documentation with composer but trying to go to http: // myproject / user I get the error:

                    Controller:
                    laminasuser (resolves to invalid controller class or alias: laminasuser)

Could you help me please.

Thank you,
                  jorge.

**Comment by fgsl** [comment]
Hi, @jmpavan. Could you paste the content of your config files here?

**Comment by jmpavan** [comment]
Yes, I'm attaching them.


Jorge Mario Paván
+54 9 11 5327 9247
jorge.pavan@pavanconsultores.com.ar

[pavan.jpg]

Sobre 12/07/2020 08:36:22, Flávio Gomes da Silva Lisboa <notifications@github.com> escribió:
Hi, @jmpavan [https://github.com/jmpavan]. Could you paste the content of your config files here?

**Comment by tiptone** [comment]
I've had the same error on a clean install/config. I eventually got past the error by updating `config/module.config.php` replacing all instances of `'controller' => 'laminasuser',` with `'controller' => 'LaminasUser',`. I also had to update `src/Controller/UserController.php` to update all `const ROUTE_` constants from `LaminasUser/*` to `laminasuser/*`.

The module still isn't working for me after that, but I got past the "resolves to invalid controller class" error and it now displays the login page.
